Vagrantfile

```diff
@@ -6,6 +6,16 @@ Vagrant.configure("2") do |config|
 
   config.vm.network "forwarded_port", guest: 80, host: 8000
 
+ # Provider for Docker, necessary for M1 chips
```

Comment:
@MattiSG Thanks for having updated the `Vagrantfile` for users owning the Apple M1 CPU!

However on my machine that change made Vagrant unable to use Virtualbox, so I had to add that line: https://github.com/openfisca/openfisca-ops/pull/95/commits/671f4c349826e6a0335ac3499f7c9d60d4c0ef6b